src/BaseTemplates/DNXEmptyWeb.vstemplate

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
-    <Name>ASP.NET 5 Empty Web Application</Name>
-    <Description>A project for creating an empty ASP.NET 5 web application</Description>
+    <Name>ASP.NET Core 1 Empty Web Application</Name>
```

Comment:
If these names show up in the New Project dialog then these shouldn't have versions in them.

cc @DamianEdwards